helm_chart/templates/ClusterRoleBinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "permission-manager.serviceAccountName" . }}
+roleRef:
+  kind: ClusterRole
+  name: {{ include "permission-manager.serviceAccountName" . }}
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "permission-manager.serviceAccountName" . }}
+    namespace: {{ .Values.config.namespace }} 
```

Comment:
any particular reason to pass it like this and not through `.Release.Namespace` (see [here](https://helm.sh/docs/chart_template_guide/builtin_objects/))

Reply:
No, let me fix that 👍

Reply:
Thank you @phisco it fixed, i also fixed the secret name which it was hardcoded